### DISCUSSION TITLE
"transpilePackages" not allowed properties on "experimental" in Nextjs 13.0.8-canary.2

### DISCUSSION THREAD

**Discussion by barayuda** [vercel/next.js#44215] [Help] [answered]
Dear Nextjs Community,
I tried canary release, with version Nextjs **13.0.8-canary.2**, with next.config.js like below:

```javascript
const path = require("path");

module.exports = {
  reactStrictMode: true,
  output: "standalone",
  experimental: {
    appDir: true,
    outputFileTracingRoot: path.join(__dirname, "../../"),
    transpilePackages: ["ui"],
  },
};

```

and, got this error message on my console:

`
The value at .experimental has an unexpected property, transpilePackages, which is not in the list of allowed properties (adjustFontFallbacks, adjustFontFallbacksWithSizeAdjust, amp, cpus, craCompat, disableOptimizedLoading, disablePostcssPresetEnv, esmExternals, appDir, externalDir, fallbackNodePolyfills, fetchCache, forceSwcTransforms, fullySpecified, gzipSize, incrementalCacheHandlerPath, isrFlushToDisk, isrMemoryCacheSize, largePageDataBytes, legacyBrowsers, manualClientBasePath, middlewarePrefetch, modularizeImports, newNextLinkBehavior, nextScriptWorkers, optimizeCss, optimisticClientCache, outputFileTracingRoot, outputFileTracingIgnores, pageEnv, profiling, proxyTimeout, runtime, serverComponentsExternalPackages, scrollRestoration, sharedPool, sri, swcFileReading, swcMinify, swcMinifyDebugOptions, swcPlugins, swcTraceProfiling, urlImports, enableUndici, workerThreads, fontLoaders, webVitalsAttribution, mdxRs, turbotrace).
`
since I read from this MR: https://github.com/vercel/next.js/pull/44194
the `transpilePackages`  move to stable, and I put config outside `experimental` but the `ui` packages not loaded to the project.
Anyone have same issue like mine? 

thanks in advance

**Comment by ciruz** [comment, marked as answer]
The error and the commit suggests that you should move this property outside experimental, like so:

```javascript
const path = require("path");

module.exports = {
  reactStrictMode: true,
  output: "standalone",
  experimental: {
    appDir: true,
    outputFileTracingRoot: path.join(__dirname, "../../"),
  },
  transpilePackages: ["ui"],
};
```

**Reply by barayuda** [reply]
I have todo that @ciruz , but my tailwind config and style inside "ui" not loaded well, it is bug?

**Reply by ciruz** [reply]
Do you have a reproduction repo with this error?

**Reply by barayuda** [reply]
all good after update to version **Nextjs version 13.1** with "transpilePackages" move outside "experimental"
thanks for reply my discussion @ciruz
